Dockerfile

```
@@ -0,0 +1,77 @@
+FROM rocker/tidyverse:4.0.0
+MAINTAINER ccdl@alexslemonade.org
+WORKDIR /rocker-build/
+
+### Install apt-getable packages to start
+#########################################
+RUN apt-get update && apt-get install -y --no-install-recommends apt-utils dialog
+
+# Add curl, bzip2 and some dev libs
+RUN apt-get update -qq && apt-get -y --no-install-recommends install \
+    curl \
+    bzip2 \
+    zlib1g \
+    libbz2-dev \
+    liblzma-dev \
+    libreadline-dev
+
+# libmagick++-dev is needed for coloblindr to install
+RUN apt-get -y --no-install-recommends install \
+    libgdal-dev \
+    libudunits2-dev \
+    libmagick++-dev
+
+# Install pip3 and instalation tools
+RUN apt-get -y --no-install-recommends install \
+    python3-pip  python3-dev
+RUN pip3 install "setuptools==46.3.0" "six==1.14.0" "wheel==0.34.2"
```

More version updates
```suggestion
RUN pip3 install "setuptools==49.3.0" "six==1.15.0" "wheel==0.34.2"
```